section2/exercises/if-statements.js

```diff
@@ -43,7 +43,13 @@ Log to the console the appropriate age range
 
 var dogAge = 3;
 // Write your conditional here
-
+if (dogAge <= 2) {
+  console.log('puppy');
+} else if (dogAge >= 3 || dogAge <= 9) {
```

Comment:
Really, we don't need to check for dogAge greater than or equal to 3 here, because we already checked for less than or equal to 2 on line 46, so any value that makes it to line 48 *must be* greater than or equal to 3. It still works, but something to think about for the future!